packages/apt/test.sh

```diff
@@ -181,7 +189,26 @@ fi
 
 echo "::endgroup::"
 
-if [ "${can_upgrade}" = "yes" ]; then
+if [ "${is_first_release}" = "no" ]; then
+  run_test
+fi
+
+echo "::group::Downgrade"
+
+if [ "${is_first_release}" = "no" ]; then
+  createdb downgrade
+  psql downgrade -c 'CREATE EXTENSION pgroonga'
+  psql downgrade -c \
+       "ALTER EXTENSION pgroonga UPDATE TO '${pgroonga_latest_released_version}'"
+  apt install -V -y --allow-downgrades \
+      ${pgroonga_package}=${pgroonga_latest_released_version}-${pgroonga_release_number}
+else
+  echo "Skip because ${pgroonga_package} hasn't been released yet."
+fi
```

Comment:
条件を反転させたほうが読みやすくない？

```shell
if [ "${is_first_release}" = "yes" ]; then
  echo "Skip because ${pgroonga_package} hasn't been released yet."
else
  createdb ...
  # ...
fi
```

Reply:
たしかに。
条件を反転させて書き直しました。